container/dockershell.sh

```diff
@@ -145,6 +150,11 @@ if [ -z ${DEPLOYMENT_TYPE} ]; then
     DEPLOYMENT_TYPE=$defaultDeployType
 fi
 
+if [ -z ${ES_VERSION} ]; then
+    printf "Event Store version (--es-version) not specified, please provide"
```

Comment:
No we want this to default to latest though ... otherwise they will always need to supply a version here.

This should be seamless for most users who do not care as they're using the latest release, so we should default to that.

Reply:
resolved by setup-container supplying current release if no version supplied